Formula/helmfile.rb

```diff
@@ -0,0 +1,29 @@
+class Helmfile < Formula
+  desc "Deploy Kubernetes Helm Charts"
+  homepage "https://github.com/roboll/helmfile"
+  url "https://github.com/roboll/helmfile/archive/v0.18.0.tar.gz"
+  sha256 "6fc6ab175a746f895b33d13f169293f55425bdc4efad33a25cf226b14b2e6f6f"
+
+  depends_on "go" => :build
+  depends_on "kubernetes-helm"
+
+  def install
+    ENV["CGO_ENABLED"] = "0"
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/roboll/helmfile").install buildpath.children
+    cd "src/github.com/roboll/helmfile" do
+      system "go", "build", "-ldflags", "-X main.Version=v#{version}", "-v", "github.com/roboll/helmfile"
+      bin.install "helmfile"
```

Comment:
You'll likely need to add a `prefix.install_metafiles` line here.

This ensures things like `README`, `LICENSE` and `CHANGELOG` where available are installed into the prefix. Normally Homebrew handles this automatically with a little magic, but when you relocate the entire contents of the `buildpath` to another directory for the install process as is common with `go`-using formulae that logic falls on its face & it needs a little manual nudging. It's a known issue.

Reply:
Fixed. Thanks!